packages/neon/neon_news/lib/widgets/feeds_view.dart

```diff
@@ -90,6 +89,8 @@ class NewsFeedsView extends StatelessWidget {
                     ),
                   );
                 },
+                tooltip: AppLocalizations.of(context).newsShowFeedErrorMessage,
```

Comment:
Huh what's this.

An IconButton with a text child?
A text will automatically be added to the a11y tree so we might have both the tooltip and the text.

Maybe this shouldn't be an IconButton altogether :thinking:

Reply:
I'd leave it as is right now since I'm only trying to add the tooltips for all iconbuttons and not fix everything :)